ui-test/statusBar.ts

```diff
@@ -0,0 +1,71 @@
+import { EditorView, StatusBar, VSBrowser, WebElement, Workbench } from "vscode-extension-tester";
+import { expect } from "chai";
+import { is_scarb_available } from "../test-support/scarb";
+import * as path from "path";
+
+describe("Status bar test", function () {
```

Comment:
```suggestion
describe("status bar", function () {
```

the whole idea is that all `describe` + `it` calls in the stack form a proper English sentence describing what behaviour is tested